packages/salesforcedx-sobjects-faux-generator/src/describe/sObjectDescribe.ts

```diff
@@ -289,12 +288,28 @@ export class SObjectDescribe {
     nextToProcess: number
   ): Promise<SObject[]> {
     try {
+      const org = await Org.create({
+        aliasOrUsername: await ConfigUtil.getUsername(projectPath)
+      });
+
       if (!this.accessToken || !this.instanceUrl) {
-        await this.getConnectionData(projectPath);
+        await this.getConnectionData(org);
+      }
+
+      let response: XHRResponse;
+      let options: XHROptions;
+      try {
+        options = this.buildXHROptions(types, nextToProcess);
+        response = await this.runRequest(options);
+      } catch (e) {
+        if (e.status !== 401) {
+          throw e;
+        }
+        await this.getConnectionData(org, true);
+        options = this.buildXHROptions(types, nextToProcess);
+        response = await this.runRequest(options);
       }
```

Comment:
First attempt to run the request with the current token. If it's expired, refresh it.